### ISSUE TITLE
Proc macros are autocompleted without `!`

### ISSUE THREAD

**Issue by mili-l** [intellij-rust/intellij-rust#8196]
## Environment

* **IntelliJ Rust plugin version:** 0.4.161.4270-213-nightly
* **Rust toolchain version:** 1.58.0-nightly (1af55d19c 2021-10-19) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2021.3 RC Ultimate Edition (IU-213.5744.202)
* **Operating system:** macOS 11.5.1
* **Macro expansion engine:** new
* **Name resolution engine:** new
* **Additional experimental features:** org.rust.cargo.features.settings.gutter, org.rust.cargo.evaluate.build.scripts, org.rust.macros.proc

## Problem description
Macros are autocompleted with `()` instead of `!`

## Steps to reproduce
1. There is a piece of code
Cargo.toml
```
[dependencies]
yew = { git = "https://github.com/yewstack/yew/" }
```
main.rs
```
fn main() {
    yew::/*caret*/
}
```
2. Press Ctrl+Space
3. Choose `html!`

## Expected Results
```rust
 yew::html!
```

## Actual Results
```rust
yew::html()
```
![macrosCompl](https://user-images.githubusercontent.com/1213122/143866883-11aef92c-725a-4d2f-ad19-9910a9a2fa6f.gif)

**Comment by Zymlex** [comment]
Same bug.

**Comment by dima74** [comment]
Fixed by #9313
